lib/xcactivitylog/objects.rb

```diff
@@ -258,6 +258,9 @@ class IDEActivityLogAnalyzerControlFlowStepMessage < IDEActivityLogAnalyzerStepM
   class IDEActivityLogAnalyzerWarningMessage < IDEActivityLogMessage
     attributes.freeze
   end
+  class IDEDiagnosticActivityLogMessage < IDEActivityLogMessage
```

Comment:
NIT: empty line above?
```suggestion

  class IDEDiagnosticActivityLogMessage < IDEActivityLogMessage
```